jvcl/run/JvGIFCtrl.pas

```diff
@@ -91,6 +91,7 @@   TJvGIFAnimator = class(TJvImageControl)
     destructor Destroy; override;
 
     function GetFrameBitmap(Index: Integer; var TransColor: TColor; Bitmap: TBitmap): Boolean; overload;
+    procedure SetAnimateIntervalForAll(Interval: Word);
```

Comment:
I'd rather have it named `SetAllFramesAnimateInterval` as it makes it clearer that it is working on all frames